### ISSUE TITLE
Unable to see comment without creating account on disqus.

### ISSUE THREAD

**Issue by agturcz** [FactorioBlueprints/factorio-prints#368]
Under the blueprint I've added I see someone has commented:
![image](https://user-images.githubusercontent.com/557490/146547041-b2890167-1ce3-469a-9f76-8d7e991db023.png)

However, I cannot even see the comment without creating disqus account, which I am not very keen to do, because of their greediness for personal data they do not need to have.

Please consider replacing disqus with another solution.

**Comment by Jan02** [comment]
here is a similar list https://alternativeto.net/software/disqus/

**Comment by FactorioBlueprints** [comment]
Agreed with this idea. Another problem with Disqus is that the author needs to comment or reply on their own blueprint in order to start getting email notifications about other comments.
